src/JMS/Serializer/XmlSerializationVisitor.php

```diff
@@ -44,10 +44,14 @@ class XmlSerializationVisitor extends AbstractVisitor
     private $nullWasVisited;
     private $objectMetadataStack;
 
+    /** @var boolean */
+    private $formatOutput;
```

Comment:
It's default `true`, setted on constructor. It's default `true` because it was `true` before.